lib/v2/datasource/remote/api/signup_repository.dart

```diff
@@ -97,19 +97,23 @@ class SignupRepository extends EosRepository with NetworkRepository {
         ]
         ..data = {
           'account': accountName,
-          'publicKey': publicKey,
+          'publicKey': publicKey.toString(),
           'invite_secret': inviteSecret,
           'fullname': displayName,
         }
     ];
 
     final transaction = Transaction()..actions = actions;
 
-    return buildEosClient()
-        .pushTransaction(transaction, broadcast: true)
-        .then((dynamic response) => mapEosResponse(response, (dynamic map) {
-              return response['transaction_id'];
-            }))
-        .catchError((error) => mapEosError(error));
+    try {
```

Comment:
is this crashing?

Reply:
No, but it wasn't catching errors on `buildEosClient`. So I decided to use `try-catch`

Reply:
@fbarzin I see displayName is nullable. 

These EOS transactions fail if you send null in the data, try send an empty string instead a null.

This happened to me when also in the transaction to update the profile.

With some lucky that is the mistake.

Reply:
it is. @RaulUrtecho  changed this recently in a pr as well.

Reply:
oh that was raul hehe